src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/viewtakes/view/ViewTakesView.kt

```diff
@@ -59,101 +74,121 @@ class ViewTakesView : View() {
                     })
             )
         }
-        vbox {
+        hbox {
             anchorpaneConstraints {
                 leftAnchor = 0.0
                 rightAnchor = 0.0
-                topAnchor = 0.0
                 bottomAnchor = 0.0
+                topAnchor = 0.0
             }
-            hbox {
-                addClass(ViewTakesStyles.headerContainer)
-                // Title label
-                label(viewModel.titleProperty) {
+            vbox {
```

Comment:
I believe the vbox here is the only thing inside the hbox?

Reply:
will get changed in #415